apigenerator/b_Workers/DatabaseFilesWorker.py

```
@@ -28,28 +28,28 @@ def install_database_files(result_full_path, db, script_absolute_path, db_authen
 
     if db == 'mysql':
         if 'ssh' in db_authentication_method:
-            append_database_library_to_requirements_file(os.path.join(result_full_path, "requirements.txt"), "pymysql\nsshtunnel")
+            append_database_library_to_requirements_file(os.path.join(result_full_path, "requirements.txt"), "pymysql==1.1.0\nsshtunnel==0.4.0")
```

Ruff (0.8.2)

`append_database_library_to_requirements_file` may be undefined, or defined from star imports

(F405)